infrastructure/pkgcc.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/sh -l
```

Comment:
As Brenda would say, "Not another one!"

```
curl -fsSL https://raw.githubusercontent.com/hyperledgendary/package-k8s-chaincode-action/main/pkgk8scc.sh -o pkgk8scc.sh && chmod u+x pkgk8scc.sh
```

Also... https://github.com/hyperledger/fabric-cli/issues/39